person.rb

```diff
@@ -7,10 +7,11 @@ def initialize(id, age, name = 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   attr_reader :id
-  attr_accessor :age, :name
+  attr_accessor :age, :name, :rentals
```

Comment:
- [ ] KIndly as suggested by the standards, your `getter` and `setter` I mean `attr_accessor` and `attr_reader` must always come on top of the class and not in the middle as you did. Please correct that